s2and/extentions/dataset_embedding.py

```diff
@@ -88,4 +99,4 @@ def parse_jsonl(path):
             model_input = title + '[SEP]' + abstract
             embeddings[paper_id] = model.forward(model_input)
         with open(join(embeddings_dir, dataset_name, f'{dataset_name}_embeddings.json'), 'w') as f:
-            json.dump(embeddings, f)
\ No newline at end of file
+            json.dump(embeddings, f)
```

Comment:
"f" is to generic. Also, when I worked on matching Wiley's to Atypon tags, I tried to store and load the embeddings as you did. However, when I increased the embedding length and/or the dataset size, my script crashed with SIGTERM after some minutes in the storing of a huge dict(like you I save embeddings in a file). A solution proposed in StackOverflow and in other forums was to store the embeddings in a JSONL file.

Reply:
Correct, i intend to do the improvement later since I don't work with embeddings in this branch.